core-bundle/src/ContaoManager/Plugin.php

```diff
@@ -48,6 +49,7 @@ public function getBundles(ParserInterface $parser): array
             BundleConfig::create(KnpTimeBundle::class),
             BundleConfig::create(SchebTwoFactorBundle::class),
             BundleConfig::create(CmfRoutingBundle::class),
+            BundleConfig::create(Terminal42ServiceAnnotationBundle::class),
```

Comment:
Just a minor nitpick: the class should be named `ServiceAnnotationBundle`, as `Terminal42` is already in the namespace. Symfony does not call their `RouteCollection` class `SymfonyRouteCollection`, either.

Reply:
Core bundles are different. It's `Knp\TimeBundle` and `Scheb\TwoFactorBundle` 😉

Reply:
It's also `Contao\CoreBundle` by the way 😂

Reply:
Exactly. Not `Knp\KnpTimeBundle` and not `Scheb\SchebTwoFactorBundle`. You just proved my point. 😄

Reply:
Uhm… no?

Reply:
Yes, you did. Spot the difference:

* `Knp\TimeBundle`
* `Scheb\TwoFactorBundle`
* `Contao\CoreBundle`
* `Terminal42\Terminal42ServiceAnnotationBundle`

See?

Reply:
namespace `Knp\TimeBundle` => class `KnpTimeBundle`
namespace `Contao\CoreBundle` => class `ContaoCoreBundle`
namespace `Terminal42\ServiceAnnotationBundle` => class `Terminal42ServiceAnnotationBundle`

Reply:
Hm, you are right. Sorry.